PHP 7.4 support

For now, build fails

```
/dev/shm/BUILD/php74-php-pecl-seasclick-0.1.0/NTS/SeasClick.cpp:119:30: error: 'ZEND_ACC_IMPLICIT_PUBLIC' was not declared in this scope
     SeasClick_ce->ce_flags = ZEND_ACC_IMPLICIT_PUBLIC;
                              ^~~~~~~~~~~~~~~~~~~~~~~~

```

--- a/php7_wrapper.h
+++ b/php7_wrapper.h
@@ -15,6 +15,11 @@
   | Author:  SeasX Group <ahhhh.wang@gmail.com>                          |
   +----------------------------------------------------------------------+
 */
+// PHP7.4 + 
+#if !defined(ZEND_ACC_IMPLICIT_PUBLIC)
+# define ZEND_ACC_IMPLICIT_PUBLIC ZEND_ACC_PUBLIC
+#endif
+
 // PHP7+
 #if PHP_MAJOR_VERSION < 7
 

